source/elements/oneDNN/include/dnnl_sycl.hpp

```diff
@@ -0,0 +1,237 @@
+/*******************************************************************************
+* Copyright 2016-2020 Intel Corporation
```

Comment:
2020 only

Reply:
btw, also we might want to check what to do with copyrights

Reply:
Updated, thanks for noticing. What do you mean by 

> what to do with copyrights

Reply:
ah, never mind (resolved)